src/test/projects/hello-world-with-processor/pom.xml

```diff
@@ -0,0 +1,79 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>fr.inria.gforge.spoon</groupId>
+  <artifactId>hello-world-with-processor</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>fr.inria.gforge.spoon</groupId>
+        <artifactId>spoon-maven-plugin</artifactId>
+				<version>3.2</version>
+				<executions>
+					<execution>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>generate</goal>
+						</goals>
+					</execution>
+					</executions>
+					<configuration>
+						<processors>
+							<processor>fr.inria.gforge.spoon.mojo.CountStatementProcessor</processor>
+							<processor>fr.inria.gforge.spoon.mojo.ProcessorWithProperty</processor>
+						</processors>
+							<processorProperties>
+								<processorProperty>
+									<name>fr.inria.gforge.spoon.mojo.ProcessorWithProperty</name>
+									<properties>
+										<property>
+											<name>methods</name>
+											<value><![CDATA[
+												{
+												<!-- Does not make any sense in the context of an application server and it can cause memory
```

Comment:
As @vmassol mentioned in https://github.com/INRIA/spoon/issues/3322#issuecomment-614065337, this shouldn't be a XML comment there, but a JSON one. Would be better to get back the final solution from https://github.com/INRIA/spoon/issues/3322#issuecomment-614067599 for the test.

Reply:
Small remark: there's no such thing as a JSON comment, which is why I removed all comments from the value and put them in XML above.

Reply:
> there's no such thing as a JSON comment,

What about JS comments? We use them for example for revapi: 
```
<analysisConfiguration><![CDATA[
            {
              "revapi": {
                "filter": {
                  // We're ignoring all API incompatibilities from XWiki dependencies (i.e. commons and rendering)
                  // since they're already covered by their own builds and we release all of them together.
                  "archives": {
                    "exclude": ["org\\.xwiki\\.commons:.*:.*", "org\\.xwiki\\.rendering:.*:.*"]
                  }
                },
                "ignore" : [
                  // Add more ignores below...
                  {
                    "code": "java.annotation.removed",
                    "old": "interface org.xwiki.xar.XarEntryTypeResolver",
                    "new": "interface org.xwiki.xar.XarEntryTypeResolver",
                    "annotation": "@org.xwiki.component.annotation.Component",
                    "justification": "Wrongly placed extension. Breakage is limited"
                  }
                ]
              }
            }
          ]]></analysisConfiguration>
```

Reply:
this is exactly the data of #3322 plus @vmassol closed it as resolved

Reply:
For revapi it’s no valid json but it works because revapi uses some lenient parser that accept invalid json.

Reply:
And also because I’ve coded our backward compatibility macro to parse invalid json too :)